### PR TITLE
Build: Lint css, html, json, md, mdx, yml files

### PR DESCRIPTION
**Pull request by ndelangen**
I found out you can lint these files pretty fast via a command, and also pre-commit

more consistency = good

### CODE REVIEW

File: docs/snippets/angular/badge-story.mdx.mdx [hunk 1]
```diff
@@ -18,14 +18,14 @@ import { Badge } from './badge.component';
 Let's define a story for our `Badge` component:
 
 <Story
-  name="positive"
-  args={{
+name="positive"
```
Comment (shilman) [inline docs/snippets/angular/badge-story.mdx.mdx:21]: ugly?

**Review by shilman — COMMENTED** [review]
Generally LGTM, tho I question the style choice on MDX

**Comment by ndelangen** [comment]
related:
https://github.com/storybookjs/storybook/issues/18556

**Comment by ndelangen** [comment]
I removed the mdx linting for now.

**Pull request merged**